ts/utils/hooks/bottomSheet.tsx

```diff
@@ -162,9 +162,11 @@ export const useIOBottomSheetModal = ({
   );
 
   useEffect(() => {
-    isScreenReaderEnabled()
-      .then(sre => setIsScreenReaderEnabled(sre))
-      .catch(_ => setIsScreenReaderEnabled(false));
+    const event = AccessibilityInfo.addEventListener(
+      "screenReaderChanged",
+      setIsScreenReaderEnabled
+    );
+    return () => event.remove();
```

Comment:
```suggestion
    const screenReaderChangedSubscription = AccessibilityInfo.addEventListener(
      "screenReaderChanged",
      setIsScreenReaderEnabled
    );
    return screenReaderChangedSubscription.remove;
```
I think the name “event“ is too generic. What do you think about a more descriptive name like the one I suggested?